vault/resource_pki_secret_backend_intermediate_set_signed.go

```diff
@@ -43,22 +61,42 @@ func pkiSecretBackendIntermediateSetSignedCreate(ctx context.Context, d *schema.
 		return diag.FromErr(e)
 	}
 
-	backend := d.Get("backend").(string)
+	backend := d.Get(consts.FieldBackend).(string)
 
 	path := pkiSecretBackendIntermediateSetSignedCreatePath(backend)
 
 	data := map[string]interface{}{
-		"certificate": d.Get("certificate").(string),
+		consts.FieldCertificate: d.Get(consts.FieldCertificate).(string),
 	}
 
 	log.Printf("[DEBUG] Creating intermediate set-signed on PKI secret backend %q", backend)
-	_, err := client.Logical().Write(path, data)
+	resp, err := client.Logical().Write(path, data)
 	if err != nil {
 		return diag.Errorf("error creating intermediate set-signed on PKI secret backend %q: %s", backend, err)
 	}
 	log.Printf("[DEBUG] Created intermediate set-signed on PKI secret backend %q", backend)
 
 	d.SetId(path)
+
+	computedIssuerFields := []string{consts.FieldImportedIssuers, consts.FieldImportedKeys}
+
+	for _, k := range computedIssuerFields {
```

Comment:
While this is a version specific field added in 1.11, since Vault returns no response on versions 1.10 and below, this was easier to handle in this way. We only set the multi-issuer fields to state if they are returned in the response, otherwise we set the computed values to nil to resolve any potential TF drift. Happy to talk more about this if folks would like